edgedb-client/src/builder.rs

```diff
@@ -1049,14 +1055,17 @@ impl Builder {
         let mut version = ProtocolVersion::current();
         let (input, output) = stream.split();
         let mut conn = Connection {
-            ping_interval: Duration::from_secs(50),
+            ping_interval: None,
             input,
             output,
             input_buf: BytesMut::with_capacity(8192),
             output_buf: BytesMut::with_capacity(8192),
             params: TypeMap::custom(),
             transaction_state: TransactionState::NotInTransaction,
-            state: State::Normal,
+            state: State::Normal {
+                #[cfg(feature="unstable")]
```

Comment:
This doesn't need to be unstable, I think you can do:
```suggestion
                #[cfg_attr(not(feature="unstable", allow(dead_code))]
```
Instead and remove all other `feature=unstable` flags except for the `ping_while` method.

Reply:
You mean putting this conditional `allow(dead_code)` at the top of the file, right? I added `unstable` because I don't want future `edgedb-rust` users to calculate the timestamps. But yeah I'm okay with removing the `unstable`s.

Reply:
No, it only needs to be on the field that is unused.